web-console/src/components/header-bar/restricted-mode/__snapshots__/restricted-mode.spec.tsx.snap

```diff
@@ -0,0 +1,69 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RestrictedMode matches snapshot 1`] = `
+<Blueprint4.Popover2
+  boundary="clippingParents"
+  captureDismiss={false}
+  content={
+    <Memo(PopoverText)>
+      <p>
+        The console is running in restricted mode.
+      </p>
+      <p>
+        It appears that you are accessing the console on the Coordinator/Overlord shared service. Due to the lack of access to some APIs on this service the console will operate in a limited mode. The unrestricted version of the console can be accessed on the Router service.
```

Comment:
```suggestion
        You are accessing the console on the Coordinator/Overlord shared service. Because this service lacks  access to some APIs, the console will operate in a limited mode. You can access the unrestricted version of the console on the Router service.
```
Are we able to provide a link to the Router service?